test/e2e/operator.go

```diff
@@ -217,6 +217,51 @@ var _ = Describe("ARO Operator - RBAC", func() {
 	})
 })
 
+var _ = Describe("ARO Operator - MachineHealthCheck", func() {
+	const (
+		mhcNamespace            = "openshift-machine-api"
+		mhcName                 = "aro-machinehealthcheck"
+		mhcRemediationAlertName = "mhc-remediation-alert"
+	)
+
+	getMachineHealthCheck := func(g Gomega, ctx context.Context) {
+		_, err := clients.MachineAPI.MachineV1beta1().MachineHealthChecks(mhcNamespace).Get(ctx, mhcName, metav1.GetOptions{})
+		g.Expect(err).ToNot(HaveOccurred())
+	}
+
+	deleteMachineHealthCheck := func(g Gomega, ctx context.Context) {
+		err := clients.MachineAPI.MachineV1beta1().MachineHealthChecks(mhcNamespace).Delete(ctx, mhcName, metav1.DeleteOptions{})
+		Expect(err).NotTo(HaveOccurred())
+	}
+
+	getMachineHealthCheckRemediationAlertName := func(g Gomega, ctx context.Context) {
+		_, err := clients.Monitoring.MonitoringV1().PrometheusRules(mhcNamespace).Get(ctx, mhcRemediationAlertName, metav1.GetOptions{})
+		g.Expect(err).ToNot(HaveOccurred())
```

Comment:
What's the difference between `g.Expect(err).ToNot(HaveOccurred())` and `Expect(err).NotTo(HaveOccurred())`?

Reply:
Not sure, but the [documentation of Gomega](https://onsi.github.io/gomega/#category-3-making-assertions-eminem-the-function-passed-into-codeeventuallycode) says to do it like this